aten/src/ATen/native/mkl/SpectralOps.cpp

```diff
@@ -230,9 +236,6 @@ Tensor _fft_mkl(const Tensor& self, int64_t signal_ndim,
   }
   // create descriptor with signal size
   std::vector<MKL_LONG> mkl_signal_sizes(checked_signal_sizes.begin(), checked_signal_sizes.end());
```

Comment:
Will this work if `MKL_LONG` is not the same type as `int64_t`?

Reply:
Yes. Ref: https://stackoverflow.com/a/6399098